### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by daniel-montalvo**

### CODE REVIEW

**Comment by daniel-montalvo** [comment]
Placeholders for definitions are now fixed.

File: index.md [hunk 1]
```diff
@@ -1,554 +1,361 @@
...
-
-***assistive technology***{:#defs-ats}
-:   Assistive technologies are software or equipment that people with disabilities use to improve interaction with the web, such as **screen readers**{:#defs-screenreader} that read aloud web pages for people who cannot read text, **screen magnifiers**{:#defs-screenmagnifier} for people with some types of low vision, and **voice recognition software** and **selection switches** for people who cannot use a keyboard or mouse.
+---
+title: "Better Web Browsing: Tips for Customizing Your Computer"
+permalink: /better-browsing/
+ref: /better-browsing/
+lang: en
+github:
+  repository: wai-better-browsing
```
Comment (yatil) [inline index.md:7]: `repository` should be `w3c/wai-better-browsing`

**Review by yatil — COMMENTED** [review]
Here’s some feedback I found [from May 23, 2019](https://lists.w3.org/Archives/Public/public-eo-plan/2019AprJun/0008.html) that I don’t want to be lost:

0. I generally like the sections and the organization of content.

1. I find the first level of expand/collapse that reveals only more expand/collapses not very intuitive. For keyboard users, we just create another level of interaction and for visual users the sub-levels are harder to scan. I would just remove that level of expand/collapse.

2. In the frontmatter: `repository: wai-better-browsing` -> `repository: w3c/wai-better-browsing`

3. Considering the audience (non-tech-savvy users), the content is too wordy and has a lot of jargon in it. I think it would be good to use more bullet lists and maybe highlight the most effective, most widely supported thing you can do up-front or inline.

4. The summary on top of the page is not a summary of the page but a description of the page. The summary should never start with “This document” because it is on this page, and it should not describe the document but summarize the content (hence the heading “Summary” 😉).

  For this document, I would suggest a summary along the lines of:

  > Changing settings in your browser (Edge, Chrome, Firefox, Safari, …) and operating systems (Windows, macOS, Linux, Android, iOS) can help you to have a better experience on the web.

5. Title/Content very desktop/laptop-specific. Should include mobile (or get a sub-resource for mobile). If we want it to be PC/Mac specific, let’s call it “Tips for Customizing Your PC/Mac”, I guess people search more often for PC or Mac in this context than computer. If we make it device independent, maybe “Tips for Customizing Your Device” or just “Tweak Your Browsing Experience”.

  (I find Customizing to be an unnecessarily long word, there are some [synonyms](https://www.thesaurus.com/browse/customizing?s=t) that are much shorter and easier to read, like “tweak”, ”adapt”, “adjust”.)

6. Headings should be front-loaded, maybe not questions.

7. Use a more descriptive title than “Terminology”, maybe “Terms”

8. I don’t see a lot of use for the lengthy introduction. Maybe we can cut to the chase quicker.

9. Some keyboard functionality is Windows-specific. Keyboard keys should be noted with the `<kbd>` element. Key combinations noted with a `+`,  so instead of `“Alt Arrow-Left”`, it should be: `<kbd>Alt</kbd> + <kbd>Arrow-Left</kbd>` or `<kbd>Alt</kbd> + <kbd>Left Arrow</kbd>` or, which I would prefer: `<kbd>Alt</kbd> + <kbd>  ⃪ </kbd>`.

10. Unsure why this sentence is needed: “Note: Laptop users can attach external mice in addition to the pointing device built into the computer.” – You can attach a mouse to any computer, including laptops.

11. Unsure if we need to define “pointing devices” here, I think it introduces jargon. Also, many laptops have touch input now, so we should at least mention this.
